spec/features/user_signup_functionality_spec.rb

```diff
@@ -1,6 +1,6 @@
 require "rails_helper"
 
-describe 'a new user signin up', type: :feature do
+describe 'a new user signs up', type: :feature do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.